lib/i18n/tasks/scanners/erb_ast_scanner.rb

```diff
@@ -3,7 +3,6 @@
 require "i18n/tasks/scanners/ruby_scanner"
 require "i18n/tasks/scanners/local_ruby_parser"
 require "i18n/tasks/scanners/occurrence_from_position"
-require "prism"
```

Comment:
this line made my i18n-task fail like this:

<img width="2708" height="220" alt="image" src="https://github.com/user-attachments/assets/b6ad513c-af89-43c2-9f4a-5b133d55b6d9" />

Reply:
Ah, will make a fix for it.

Reply:
https://github.com/glebm/i18n-tasks/pull/689